extensions/scc/scc.go

```diff
@@ -0,0 +1,14 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package extscc
```

Comment:
We should also eventually support ucc. I assume you are thinking this would be a different package/mechanism (as this has "s"cc)?

Reply:
Good point. Maybe I should rename this package to chaincode, to leave it open to adding UCC support later.